### DISCUSSION TITLE
Cat Voices

### DISCUSSION THREAD

**Discussion by EvilIrving** [creatorsand/.github#76] [Ideas]
一句话 IDEA：

录下猫猫**打呼噜**、**讨好你**、**惊吓**、**发嗲**等等状态时的声音，无限循环回放给猫猫

**我的角色**：

开发写代码

**其他角色需求**：

设计，用研

**社群昵称**：

Cain

**个人简介**：

摸索学习的 new swift developer。

**产品是给谁用的？**

出于对猫的爱想要记录声音的铲屎官

**其他产品有什么痛点，用户为什么要选择你的产品？**

市面上的太重了，功能多而繁杂，本产品专注记录猫咪声音

**Comment by ShirleneLiu** [comment]
所以这个 app 的用户是猫猫吗？😂

**Reply by EvilIrving** [reply]
当然是铲屎官啦，铲屎官只能寄希望于猫猫积极参与！

**Comment by ZigaoWang** [comment]
作为一个铲屎官，我很期待！
